CHANGELOG/CHANGELOG-1.3.md

```diff
@@ -31,6 +31,7 @@
 - Do not run controller service in node. ([#702](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/702), [@mattcary](https://github.com/mattcary))
 - Fix a bug that CreateVolume should round up the request_bytes. ([#684](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/684), [@Jiawei0227](https://github.com/Jiawei0227))
 - It is now possible to mount a volume with XFS filesystem and its restored snapshot. ([#788](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/788), [@jsafrane](https://github.com/jsafrane))
+- Fix a bug where ControllerPulishVolume is repeatedly called when the gke Controller is in different zone with gke nodes ([#813](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/813), [@leiyiz](https://github.com/leiyiz))
```

Comment:
Drop the CHANGELOG change, this is done automatically by picking up the release-notes from each PR.